packages/fxa-payments-server/src/routes/Subscriptions/ActionButton.tsx

```diff
@@ -0,0 +1,100 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { Localized } from '@fluent/react';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { Customer } from '../../store/types';
+
+export type ActionButtonProps = {
+  customer: Customer;
+  onRevealUpdateClick: () => void;
+  revealFixPaymentModal: () => void;
+};
+
+export const ActionButton = ({
+  customer,
+  onRevealUpdateClick,
+  revealFixPaymentModal,
+}: ActionButtonProps) => {
+  const {
+    billing_agreement_id,
+    payment_provider,
+    paypal_payment_error,
+  } = customer;
+
+  const stripeActionButton = (
+    <button
+      data-testid="reveal-payment-update-button"
+      className="settings-button"
+      onClick={onRevealUpdateClick}
+    >
+      <Localized id="pay-update-change-btn">
+        <span className="change-button" data-testid="change-button">
+          Change
+        </span>
+      </Localized>
+    </button>
+  );
```

Comment:
nit: this causes React.createElement to be called immediately regardless we end up using the Stripe button or not.  We could have it as a function, `const stripeActionButton = () => (...` then call it where it's used.

Reply:
What was the approach before that, Dave/Barry? I don't remember, and it'd be good to keep it in mind.

Reply:
@biancadanforth Not sure what you are asking here, do you mean what was the code before I addressed your comment and turned the variables into functions instead of element declarations?

Reply:
Exactly @meandavejustice -- how were the components declared before?

Reply:
@biancadanforth They were just about the same, but without being wrapped in a function.

```
const stripeActionButton = (
    <button
      data-testid="reveal-payment-update-button"
      className="settings-button"
      onClick={onRevealUpdateClick}
    >
      <Localized id="pay-update-change-btn">
        <span className="change-button" data-testid="change-button">
          Change
        </span>
      </Localized>
    </button>
  );
```